CHARTER.md

```diff
@@ -0,0 +1,101 @@
+# DevRel Foundation Structure and Organization Working Group Operating Charter (DRAFT)
+
+## 1. Working Group Definition
+
+### Tl;Dr:
+The primary objective of this working group is to document best practices for structuring and organizing Developer Relations (DevRel) teams within companies, including:
+- Team size
+- Roles and responsibilities
+- Reporting structure
+
+Developer Relations teams vary significantly across organizations, influenced by company size, customer personas, and strategic priorities. The DevRel Foundation’s Structure and Org Working Group aims to identify and categorize the different models of DevRel teams, understanding how these factors shape their function. This working group will produce resources that help organizations define, evaluate, and optimize their DevRel teams to align with business goals.
+
+## 2. Scope & Objectives
+
+The working group focuses on:
+- Identifying the different organizational models of DevRel teams across industries.
+- Documenting best practices for structuring DevRel teams, including roles, responsibilities, and team size.
+- Understanding how DevRel reporting structures impact effectiveness.
+- Analyzing the impact of company size, industry, and customer persona on DevRel team design.
+- Creating resources that help organizations evaluate and optimize their DevRel team structure.
```

Comment:
Might this include something along the lines of job descriptions, interview questions / procedures, etc?